tests/installation/network_configuration.pm

```diff
@@ -0,0 +1,40 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2018 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Verify default options in Network Configuration during installation
+# and modify some of these options.
+# Maintainer: Joaquín Rivera <jeriveramoya@suse.de>
+
+use base "y2logsstep";
+use strict;
+use warnings;
+use testapi;
+use utils 'assert_screen_with_soft_timeout';
+
+sub run {
+    if (!get_var("HDD_SCC_REGISTERED")) {
+        assert_screen_with_soft_timeout(
+            'scc-registration',
+            timeout      => 350,
+            soft_timeout => 300,
+            bugref       => 'bsc#1028774'
+        );
+    }
```

Comment:
where did you get this block from?

Reply:
scc_registration.pm

Reply:
ok, better don't duplicate but extract method